### DISCUSSION TITLE
Create a shortcut to select a Bible passage by typing it

### DISCUSSION THREAD

**Discussion by entodoays** [ezra-bible-app/ezra-bible-app#1166] [Ideas]
For those features that I use frequently in any programme, I prefer using keyboard shortcuts. This doesn't seem to be possible in Ezra.
One neat shortcut I use in Xiphos is Ctrl + L to cho0se a bible passage by typing it, instead of clicking buttons. I recommend implementation of keyboard shortcuts.

**Comment by tobias-klein** [comment]
@entodoays Thanks for raising this! It is indeed something that has been considered and discussed before, see #701. I could try to make it happen in one of the next releases!
